internal/config/canary_config.go

```diff
@@ -91,7 +90,7 @@ const (
 type DynamicCanaryConfig struct {
 	SaramaLogEnabled      *bool `json:"saramaLogEnabled"`
 	VerbosityLogLevel     *int  `json:"verbosityLogLevel"`
-	ActivateTracing       *bool //if Jaeger or OTLP env var are defined we activate Tracing
+	OtlpExporterTracing   *bool //if Jaeger or OTLP env var are defined we activate Tracing
```

Comment:
What was your thinking behind re-naming the variable/flag? I thought the point of the change was to allow us to swap between tracing being active and in-active at will without the instrumented code having to be aware of the change. Therefore the old name perfectly encapsulated the intent for me.

Reply:
traces are activated if JAEGER or OTLP env are defined. This bool was not needed but i can rollback it as you prefer

Reply:
I liked the idea of separating tracing being turned on or off from the configuration of the endpoint but that's my personal preference.


@ppatierno do you have a preference?

Reply:
I put the activateTracing bool. the dynamicconfig does not work as we have to get a new producer and a new consumer to update the tracing.

Reply:
please tell me what is needed to validate this PR :)